app/models/host/managed.rb

```diff
@@ -415,6 +415,7 @@ def self.import_host_and_facts(hostname, facts, certname = nil, proxy_id = nil)
     # downcase everything
     hostname.try(:downcase!)
     certname.try(:downcase!)
+    facts['domain'].try(:downcase!)
```

Comment:
I thought of the same thing, however hostname and certname are utilizing try, so I was just keeping it consistent.  Let me know if the change is necessary.